src/components/Header/Header.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  cleanup,
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import { AuthProvider } from 'context/auth';
+import Header from 'components/Header/Header';
+import { mockServer } from 'mock/mockServer';
+
+if (process.env.REACT_APP_MOCK_API_TRUE) {
+  mockServer();
+}
+
+// Fn's to resize screen
+const resizeToMobile = () => {
+  global.innerWidth = 600;
+  global.dispatchEvent(new Event('resize'));
+};
+
+const resizeToDesktop = () => {
+  global.innerWidth = 900;
+  global.dispatchEvent(new Event('resize'));
+};
+
+describe('<Header />', () => {
+  beforeEach(() => {
+    render(
+      <AuthProvider>
+        <Header />
+      </AuthProvider>
+    );
+
+    cleanup;
```

Comment:
put this in aftereach as a callback

afterEach(cleanup);

Reply:
fixed in 7e6c6003d60350fc9b2eb63ec1aecf1599f3dfdc